tooling/minitest/src/ub/spawn.rs

```diff
@@ -41,13 +39,10 @@ fn spawn_arg_count() {
             next_block: Some(BbName(Name::from_internal(1))),
         }
     );
-    
     let b1 = block!(exit());
-    
-    let f = function(Ret::No, 0, &locals, &[b0, b1]);
-    
+
+    let f = function(Ret::No, 0, &[], &[b0, b1]);
```

Comment:
In terms of paragraph structure, I would put the `function` together with its basic blocks. Applies to all tests you changed here.